alma/alma-installments-prestashop
# fix: Banner displays another message when account is linked

### Reason for change



[Linear task](https://linear.app/almapay/issue/ECOM-1811/[💅-ps]-banner-displays-another-message-when-account-is-linked)

### Code changes


When the merchant has not yet link their PS Account : nothing to be updated.
When the merchant has linked their PS account in order to use Alma
And When the merchant is on Alma's configuration's page

### How to test

_As a reviewer, you are encouraged to test the PR locally._


Install the Alma module and see the user flow with PS Account

### Checklist for authors and reviewers




### Non applicable
